Makefile

```diff
@@ -191,6 +191,10 @@ RELEASE_TAG := $(shell git describe --abbrev=0 2>/dev/null)
 .PHONY: release
 release:  ## Builds and push container images using the latest git tag for the commit.
 	@if [ -z "${RELEASE_TAG}" ]; then echo "RELEASE_TAG is not set"; exit 1; fi
+	@echo "!!! WARNING !!!\nThis commands cleans all local changes and switches the current working branch.\n\nDo you want to proceed? [y/N] " \
+		&& read ans && [ $${ans:-N} = y ]
+	git clean -qfdx
```

Comment:
:grimacing: I think I would rather just error out if we encounter a dirty repo rather than this.

Reply:
I don’t mind either way, @ncdc wdyt?

Reply:
+1 to error out

Reply:
Done, ptal